backend/src/main/java/com/java/academy/gym/model/Locale.java

```diff
@@ -0,0 +1,76 @@
+package com.java.academy.gym.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Entity
+@Table(name = "locales")
+public class Locale extends BaseEntity {
+
+    @NotNull
+    @Size(max = 10)
+    @Column(unique = true)
```

Comment:
Instead of or together with "@NotNull" "@Size" and so on annotations better use JPA "@Column" annotation because it affects the database table if you using tools like hbmddl that create DDL scheme. in this case it looks like @Colimn(name = "LANG_CODE", nullable = false, unique = true, length=10)